viper.go

```diff
@@ -1205,6 +1205,15 @@ func (v *Viper) IsSet(key string) bool {
 	return val != nil
 }
 
+// IsConfigured checks to see if the key has been set in any of the data locations but defaults.
```

Comment:
Minor nit: mention earlier in the doc that this is specifically about non-default values.

```suggestion
// IsConfigured checks to see if the key has a non-default value set.
```